src/components/LiquidityChartRangeInput/Zoom.tsx

```diff
@@ -135,3 +137,5 @@ export default function Zoom({
     </Wrapper>
   )
 }
+
+export default styled(Zoom)``
```

Comment:
why empty styled

Reply:
```
const CustomZoom = styled(Zoom)<{ $interactive: boolean }>`
  ${({ $interactive }) => (!$interactive ? 'top: -46px;' : '')}
`
```
Need to make it styled so the above will work